7zip/README.md

```diff
@@ -1,15 +1,20 @@
 # 7zip
 
-7-Zip is a file archiver with a high compression ratio
+7-Zip is a file archiver with a high compression ratio.
 
 ## Maintainers
 
-* The Habitat Maintainers: <humans@habitat.sh>
+The Habitat Maintainers: <humans@habitat.sh>
 
 ## Type of Package
 
-Binary package
+Binary package.
 
 ## Usage
 
-*TODO: Add instructions for usage*
+Install the package and execute 7zip:
+
+```
+hab pkg install core/7zip --binlink
```

Comment:
Unfortunately Windows does not support binlinking so this would panic there. Because this includes a windows plan this becomes relevant. We could replace these 2 lines with a `hab pkg exec` example or just note that on Windows binlinking is not supported and add the `hab pkg exec` example.

Reply:
@mwrock Fixed